lang/jmh/org/partiql/jmh/MultipleLikeBenchmark.kt

```diff
@@ -0,0 +1,403 @@
+        val data10 = """
+            { 
+                'hr': { 
+                    'employees': <<""" + employeeData10.joinToString(",") + """>>
+                } 
+            }
+        """.trimIndent()
+
+        val bindings10 = pipeline.compile(parser.parseAstStatement(data10)).eval(EvaluationSession.standard()).bindings
+        val session10 = EvaluationSession.build { globals(bindings10) }
+
+        val query15 = """
+            SELECT * 
+            FROM hr.employees as emp
+            WHERE lower(emp.name) LIKE '%bob smith%'
+               OR lower(emp.name) LIKE '%gage swanson%'
+               OR lower(emp.name) LIKE '%riley perry%'
+               OR lower(emp.name) LIKE '%sandra woodward%'
+               OR lower(emp.name) LIKE '%abagail oconnell%'
+               OR lower(emp.name) LIKE '%amari duke%'
+               OR lower(emp.name) LIKE '%elisha wyatt%'
+               OR lower(emp.name) LIKE '%aryanna hess%'
+               OR lower(emp.name) LIKE '%bryanna jones%'
+               OR lower(emp.name) LIKE '%trace gilmore%'
+               OR lower(emp.name) LIKE '%antwan stevenson%'
+               OR lower(emp.name) LIKE '%julianna callahan%'
+               OR lower(emp.name) LIKE '%jaelynn trevino%'
+               OR lower(emp.name) LIKE '%kadence bates%'
+               OR lower(emp.name) LIKE '%jakobe townsend%'
+            """
+        val astStatement15 = parser.parseAstStatement(query15)
+        val expression15 = pipeline.compile(astStatement15)
+
+        val query30 = query15 + """
+               OR lower(emp.name) LIKE '%austin pennington%'
+               OR lower(emp.name) LIKE '%colby woodward%'
+               OR lower(emp.name) LIKE '%brycen blair%'
+               OR lower(emp.name) LIKE '%cristal mercer%'
+               OR lower(emp.name) LIKE '%river gilmore%'
+               OR lower(emp.name) LIKE '%saniya bowers%'
+               OR lower(emp.name) LIKE '%braedon ross%'
+               OR lower(emp.name) LIKE '%clark mejia%'
+               OR lower(emp.name) LIKE '%ryan day%'
+               OR lower(emp.name) LIKE '%marilyn luna%'
+               OR lower(emp.name) LIKE '%avah sanchez%'
+               OR lower(emp.name) LIKE '%amelie wu%'
+               OR lower(emp.name) LIKE '%paola duke%'
+               OR lower(emp.name) LIKE '%jesse trevino%'
+               OR lower(emp.name) LIKE '%bianca cisneros%'
+            """
+        val astStatement30 = parser.parseAstStatement(query30)
+        val expression30 = pipeline.compile(astStatement30)
+    }
+
+    /**
+     * Example PartiQL benchmark for parsing a query
+     */
+    @Benchmark
+    @BenchmarkMode(Mode.AverageTime)
+    @OutputTimeUnit(TimeUnit.MICROSECONDS)
+    @Fork(value = 2)
+    fun testPartiQLParser15(state: MyState, blackhole: Blackhole) {
+        val expr = state.parser.parseAstStatement(state.query15)
+        blackhole.consume(expr)
+    }
+
+    /**
+     * Example PartiQL benchmark for compiling a query
```

Comment:
nit: sorry for missing this earlier, but would be great have the explanation of each benchmark in the kdocs.